projects/valtimo/components/src/lib/components/view-content/type-converters/dateTypeConverter.ts

```diff
@@ -28,6 +28,7 @@ export class DateTypeConverter implements TypeConverter {
     if (!value) {
       return '-';
     }
-    return moment(value).format('DD-MM-YYYY, hh:mm A');
+
+    return moment(value).format(definition.format != null && definition.format !== '' ? definition.format : 'DD-MM-YYYY, hh:mm A');
```

Comment:
`definition?.format || 'DD-MM-YYYY, hh:mm A'` should work the same

Reply:
I have applied this